data/json/items/discs.json

```diff
@@ -0,0 +1,83 @@
+[
+  {
+    "type": "GENERIC",
+    "id": "dvd_blank",
+    "name": { "str": "blank DVD disc" },
+    "description": "An optical disc of medium storage, once used for storing and playing digital video, audio, and other data.  Many years before the Cataclysm, it was a popular format for movies, software, and various multimedia content.  This one is blank, has no data on it.",
+    "category": "other",
+    "material": [ "plastic" ],
+    "weight": "16 g",
+    "volume": "7 ml",
+    "longest_side": "120 mm",
+    "price": 250,
+    "price_postapoc": 2,
+    "symbol": "O",
+    "color": "gray"
+  },
+  {
+    "type": "GENERIC",
+    "id": "dvd_scratched",
+    "name": { "str": "scratched DVD disc" },
+    "description": "An optical disc of medium storage, once used for storing and playing digital video, audio, and other data.  Many years before the Cataclysm, it was a popular format for movies, software, and various multimedia content.  This one is scratched to the point it is impossible for any device to read its contents.",
+    "copy-from": "dvd_blank",
+    "price": 5,
+    "price_postapoc": 0,
+    "flags": [ "TRADER_AVOID" ],
+  },
+  {
+    "type": "GENERIC",
+    "id": "dvd_cracked",
+    "name": { "str": "cracked DVD disc" },
+    "description": "An optical disc of medium storage, once used for storing and playing digital video, audio, and other data.  Many years before the Cataclysm, it was a popular format for movies, software, and various multimedia content.  This one is cracked and damaged to the point it is impossible for any device to read its contents.",
+    "copy-from": "dvd_blank",
```

Comment:
I am trying to avoid adding variants as much as possible, Doc.  Instead, I'd rather use copy-from, because that way I feel more comfortable dealing with item groups, as I have plans for these 'garbage items' to appear at the right places.  Also we can say that a scratched DVD disc has a higher value than a cracked one, heh.